django/test/__init__.py

```diff
@@ -7,12 +7,12 @@
 )
 from django.test.utils import (
     ignore_warnings, modify_settings, override_settings,
-    override_system_checks, tag,
+    override_system_checks, register_lookup, tag,
```

Comment:
I don't think this needs to be namespaced in `django.test`. It's not documented as a public API at least.